Test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,64 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+});
```

Comment:
Describe es una funcion que nos sirve para agrupar pruebas. Por esta razon debes tener los `it` dentro de `describe`